app/src/main/java/ch/epfl/sdp/appart/database/local/LocalDatabaseService.java

```diff
@@ -0,0 +1,127 @@
+package ch.epfl.sdp.appart.database.local;
+
+import android.graphics.Bitmap;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import ch.epfl.sdp.appart.ad.Ad;
+import ch.epfl.sdp.appart.scrolling.card.Card;
+import ch.epfl.sdp.appart.user.User;
+
+public interface LocalDatabaseService {
```

Comment:
I'm not sure of the reason of this interface. The rease we have a DatabaseService/LoginService is in case we'd want to switch to an alternative from Firebase/Firestore. But here Localbase will always be the only class to implement this interface, won't it ?

Reply:
It's for the Singleton Hilt injection